hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -161,7 +161,7 @@ Default: ""
 Version: stable/pieeprom-2020-03-19.bin  
 
 ### DISABLE_HDMI
-Disables the HDMI boot diagnostics display if a fatal error is encountered. This may also be disabled by setting display_splash=1 in config.txt.
+Disables the [HDMI boot diagnostics](./boot_diagnostics.md) display if a fatal error is encountered. This may also be disabled by setting display_splash=1 in config.txt.
```

Comment:
Was this supposed to be `disable_splash=1` ?

Reply:
Yes